packages/cli/src/commands/inspect.ts

```diff
@@ -91,9 +98,42 @@ export async function inspect(
         .map((o) => `${o[0]}=${o[1]}`)
         .join(' ') || '(none)'
     );
+    packageOwner ? console.log('           Owner:', packageOwner) : console.log('          Source:', localSource || '(none)');
     console.log('     Package URL:', deployUrl);
     console.log('        Misc URL:', deployData.miscUrl);
-    console.log('Package Info URL:', metaUrl);
+    console.log('Package Info URL:', metaUrl || '(none)');
+    console.log('Cannon generator:', deployData.generator);
+    console.log('       timestamp:', deployData.timestamp);
+    console.log('IPFS Availability Score(# of nodes): ', ipfsAvailabilityScore || 'Run IPFS Locally to get this score');
+    console.log()
+    console.log(yellow(bold('Smart Contracts')))
+    console.log(`Note: Any ${bold("contract name")} that is bolded is highlighted and marked as important.`);
+    console.log('Contract Addresses:');
+    console.log('-------------------');
+    for (let contractName in contractsAndDetails) {
+        const { address, highlight } = contractsAndDetails[contractName];
+        const displayName = highlight ? bold(contractName) : contractName;
+        console.log(`${displayName}: ${address}`);
+    }
+    console.log('-------------------');
+    console.log();
+    if (sources) {
+      console.log('Contract Sources:');
+      console.log('-------------------');
+      for (let contractName in contractsAndDetails) {
+          const { sourceName, highlight } = contractsAndDetails[contractName];
+          if (sourceName) {
+              const displayName = highlight ? bold(contractName) : contractName;
+              console.log(`${displayName}: ${sourceName}`);
```

Comment:
great that we have access to displaying the contract sources. This can be done in a future PR, but part of what I was thinking wrt this feature would be having the ability to dump a contract source, rather than printing out the normal output. for example, 

```
cannon inspect synthetix --source contracts/modules/AccountModule.sol
```

would simply print out the contents of that file. or the user can specify a address like:


```
cannon inspect synthetix --source AccountModule
```

would display *all* the contract sources corresponding to the account module.